engine/component.ftl

```diff
@@ -137,6 +137,25 @@
     }
 ] ]
 
+[#macro addResourceGroupAttributeValues type extensions provider]
+
+    [@internalMergeComponentConfiguration
+        type=type
+        configuration=
+            {
+                "ResourceGroups" : {
+                    DEFAULT_RESOURCE_GROUP : {
+                        "Extensions" : {
+                            provider :
+                                extensions
```

Comment:
There can be shared attributes for any resource group so I think this function needs an extra attribute? If you look at the modified code where the attributes are assembled, it loops through the resource groups.

Reply:
Cheers, updated in 1669096